README.md

```diff
@@ -67,7 +67,7 @@ Also you can add command `dart run layerlens --fail-on-cycles` to the repo's pre
 
 If you want to generate the `DEPENDENCIES.md` only for a specific folders, you can use `--build-filter` option and you should use [glob](https://pub.dev/packages/glob) syntax. For example, to generate the diagram only for the root `lib/` folder, you run following `dart run layerlens --build-filter "lib"`.
 
-You can specify multiple build filters . The mechanism is inspired by `--build-filter` in Dart's [`build_runner`](https://github.com/dart-lang/build/blob/master/docs/partial_builds.md). For example, to run the layerlens for root `lib/` and it's subfolder `lib/subfolder1` run `layerlens --build-filter "lib" --build-filter "lib/subfolder1"`
+You can specify multiple build filters . The mechanism is inspired by `--build-filter` in Dart's [`build_runner`](https://github.com/dart-lang/build/blob/master/docs/partial_builds.md). For example, to run the layerlens for root `lib/` and it's subfolder `lib/subfolder1` run `layerlens --build-filter "lib" --build-filter "lib/subfolder1"`. To generate the entire subtree for a given subfolder you can run following: `layerlens --build-filter "lib/subfolder1" --build-filter "lib/subfolder1/**"`
```

Comment:
Should one of them be subfolder2 instead of subfolder1?
Actually in this example it is better to have just one filter:
`layerlens --build-filter "lib/subfolder1/**"`

Reply:
If I want to cover the entire subtree of `lib/subfolder1/` including the `lib/subfolder1` itself (ie `lib/subfolder1/DEPENDENCIES.md`), I have to specify both `--build-filter "lib/subfolder1" --build-filter "lib/subfolder1/**"` to match the folders.

If I would do just `layerlens --build-filter "lib/subfolder1/**"` it would not generate `lib/subfolder1/DEPENDENCIES.md` (afaik as it should be according to the `glob` syntax, because `lib/subfolder1/**` does not match `lib/subfolder1`)

This test exactly test such scenario https://github.com/polina-c/layerlens/pull/18/files#diff-b75cc7da7efc55a935d2cf451646e4698babc43aa24af2727771f6ce8d2bb17aR256